Data/Vector/Generic.hs

```diff
@@ -424,6 +434,15 @@ drop n v = unsafeSlice (delay_inline min n' len)
   where n' = max n 0
         len = length v
 
+-- | /O(1)/ Yield all but the last @n@ elements without copying. The vector may
+-- contain less than @n@ elements in which case an empty vector is returned.
+dropEnd :: Vector v a => Int -> v a -> v a
+{-# INLINE_FUSED dropEnd #-}
+dropEnd n v = unsafeSlice 0 (max 0 (m - n')) v
```

Comment:
Here also, I think we need to add a `delay_inline` for the `max`

Reply:
Done